src/autocomplete.ts

```diff
@@ -101,6 +143,15 @@ export default class Autocomplete {
     }
   }
 
+  updateFeedbackForScreenReaders(input: ScreenReaderAccouncementConfig): void {
```

Comment:
I like the DRY approach here, making it easy to update the sr announcment.  I wonder if it would be simpler to just pass a string into here, and remove the `ScreenReaderAccouncementConfig` input type.  That way calling sites could just call the exact function they need to get the string and pass it in, rather than having different `event` types with various options.  WDYT?

Reply:
I see what you're saying. I opted for this approach because of the "new-options" event having two different types of announcements, depending on if autoselect is enabled or not. It seemed like it would get messy to keep all the different combinations of strings in autocomplete.ts when it could be in its own file.

However, I could rewrite it to be an additional function, remove the events, and call "newOptionsWithAutoselect" or something. WDYT?

Reply:
I think that would probably be an easier approach to maintain 👍

Reply:
Ok please let me know what you think of the new approach @dgreif 🙏